docs/example.rst

```diff
@@ -0,0 +1,76 @@
+
+.. code-block:: bash
+
+   $ vsg -f PIC.vhd 
+   =======================================================================================
+   File:  PIC.vhd
+   =======================================================================================
+   Phase 5 of 7... Reporting
+   Total Rules Checked: 208
+   Total Violations:    523
+   ----------------------------+------------+---------------------------------------------
+     Violation                 |  line(s)   | Solution
```

Comment:
Shouldn't the name of the first column be Rule not Violation?

Reply:
The title of the whole table should probably be Violations.

Reply:
You are correct.  Violation should be Rule.